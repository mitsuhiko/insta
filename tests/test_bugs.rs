#[test]
fn test_crlf() {
    insta::assert_snapshot!("foo\r\nbar\r\nbaz");
}

#[test]
fn test_trailing_crlf() {
    insta::assert_snapshot!("foo\r\nbar\r\nbaz\r\n");
}

#[test]
fn test_trailing_crlf_inline() {
    insta::assert_snapshot!("foo\r\nbar\r\nbaz\r\n", @r###"
    foo
    bar
    baz
    "###);
}
