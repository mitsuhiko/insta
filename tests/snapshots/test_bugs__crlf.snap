---
source: tests/test_bugs.rs
expression: "\"foo\\r\\nbar\\r\\nbaz\""
---
foo
bar
baz
